TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/integ_test/integ_test/backfill_tests.py

```diff
@@ -56,10 +56,14 @@ def setup_backfill(request):
     status_result: CommandResult = snapshot.status()
     if status_result.success:
         snapshot.delete()
-    snapshot.create(wait=True)
-    metadata.migrate()
-    backfill.start()
-    backfill.scale(units=10)
+    snapshot_result: CommandResult = snapshot.create(wait=True)
+    assert snapshot_result.success
+    metadata_result: CommandResult = metadata.migrate()
+    assert metadata_result.success
+    backfill_start_result: CommandResult = backfill.start()
+    assert backfill_start_result.success
+    backfill_scale_result: CommandResult = backfill.scale(units=10)
+    assert backfill_scale_result.success
```

Comment:
Thank you!

I'm curious how you tested this, though.  Can you provide some details on that?

Reply:
I ran the Jenkins pipeline on this branch

Reply:
To clarify - @lewijacn confirmed that the test failed at the assertions when the underlying RFS operation failed.